test/Zenith.t.sol

```diff
@@ -41,29 +38,16 @@ contract ZenithTest is Test {
         commit = target.blockCommitment(header);
     }
 
-    // cannot submit block with incorrect sequence number
-    function test_badSequence() public {
-        // change to incorrect sequence number
-        header.sequence = 100;
+    // cannot submit block with incorrect host block number
+    function test_incorrectHostBlock() public {
+        // change to incorrect host block number
+        header.hostBlockNumber = 100;
         commit = target.blockCommitment(header);
 
         // sign block commitmenet with sequencer key
         (uint8 v, bytes32 r, bytes32 s) = vm.sign(sequencerKey, commit);
 
-        vm.expectRevert(abi.encodeWithSelector(Zenith.BadSequence.selector, 1));
-        target.submitBlock(header, v, r, s, blockData);
-    }
-
-    // cannot submit block with expired confirmBy time
-    function test_blockExpired() public {
-        // change to expired confirmBy time
-        header.confirmBy = block.timestamp - 1;
-        commit = target.blockCommitment(header);
-
-        // sign block commitmenet with sequencer key
-        (uint8 v, bytes32 r, bytes32 s) = vm.sign(sequencerKey, commit);
-
-        vm.expectRevert(abi.encodeWithSelector(Zenith.BlockExpired.selector));
+        vm.expectRevert(abi.encodeWithSelector(Zenith.IncorrectHostBlock.selector));
```

Comment:
probably don't need the encode step?

Reply:
ah correct, fixed: https://github.com/init4tech/zenith/pull/48/commits/d7cf45865d3457eedb41e6c2c86f81c01743acc5